scripts/project-depcheck.ts

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ts-node
+
+import path from 'path'
+
+import depcheck from 'depcheck'
+
+const options = {
+  detectors: [
+    // the target detectors
+    depcheck.detector.requireCallExpression,
+    depcheck.detector.importDeclaration,
+  ],
+  ignoreBinPackage: false, // ignore the packages with bin entry
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)